server/scripts/db-migrations/20250810034041_activist_timestamps.up.sql

```diff
@@ -0,0 +1,22 @@
+-- *_updated timestamps may be used to resolve conflicts when merging activist records
+
+-- A sentinel value is used as the default in order to indicate unknown modification times for
+-- existing table rows in production.
+ALTER TABLE activists
+    ADD COLUMN created TIMESTAMP NOT NULL DEFAULT '1999-12-31 00:00:00',
+    ADD COLUMN phone_updated TIMESTAMP NOT NULL DEFAULT '1999-12-31 00:00:00',
+    ADD COLUMN email_updated TIMESTAMP NOT NULL DEFAULT '1999-12-31 00:00:00',
+    ADD COLUMN address_updated TIMESTAMP NOT NULL DEFAULT '1999-12-31 00:00:00',
+    ADD COLUMN coords_updated TIMESTAMP NOT NULL DEFAULT '1999-12-31 00:00:00';
+
+-- Change default for new records to CURRENT_TIMESTAMP.
+ALTER TABLE activists
+    MODIFY COLUMN created TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP;
+ALTER TABLE activists
```

Comment:
hmmm, does this mean that when an activist is created, the `phone_updated` would be set to the current timestamp even if they have no phone number? from a db perspective, i feel like having these nullable makes the most sense. but if you've already thought about this, i'm fine w/ it.